lib/rabbitmq/cli/diagnostics/commands/cipher_suites_command.ex

```diff
@@ -0,0 +1,42 @@
+## The contents of this file are subject to the Mozilla Public License
+## Version 1.1 (the "License"); you may not use this file except in
+## compliance with the License. You may obtain a copy of the License
+## at http://www.mozilla.org/MPL/
+##
+## Software distributed under the License is distributed on an "AS IS"
+## basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See
+## the License for the specific language governing rights and
+## limitations under the License.
+##
+## The Original Code is RabbitMQ.
+##
+## The Initial Developer of the Original Code is GoPivotal, Inc.
+## Copyright (c) 2007-2016 Pivotal Software, Inc.  All rights reserved.
+
+
+defmodule RabbitMQ.CLI.Diagnostics.Commands.CipherSuitesCommand do
+  @behaviour RabbitMQ.CLI.CommandBehaviour
+  use RabbitMQ.CLI.DefaultOutput
+
+  def merge_defaults(args, opts), do: {args, Map.merge(%{openssl_format: false}, opts)}
+
```

Comment:
I'd use the Erlang formatter (`def formatter(), do: RabbitMQ.CLI.Formatters.Erlang`. Right now the output without the `openssl-format` switch looks like this:

```elixir
[
{:ecdhe_ecdsa, :aes_256_gcm, :null, :sha384}
,{:ecdhe_rsa, :aes_256_gcm, :null, :sha384}
,{:ecdhe_ecdsa, :aes_256_cbc, :sha384, :sha384}
,{:ecdhe_rsa, :aes_256_cbc, :sha384, :sha384}
,{:ecdh_ecdsa, :aes_256_gcm, :null, :sha384}
...
]
```

With the Erlang formatter, users would just have to copy-paste the tuples in the configuration file.